docs/products/kafka/howto/configure-topic-tiered-storage.md

```diff
@@ -1,146 +1,123 @@
     --topic exampleTopic \
-    --partitions 1 \
+    --partitions 2 \
     --replication 2 \
     --remote-storage-enable
 ```
 
 In this example:
 
--   `demo-kafka-project` is the name of your project.
--   `demo-kafka-service` is the name of your Aiven for Apache Kafka®
-    service.
--   `exampleTopic` is the name of the topic you are creating with tiered
-    storage enabled.
--   The topic will have 1 partition and a replication factor of 2.
+- `demo-kafka-project` is the name of your project.
+- `demo-kafka-service` is the name of your Aiven for Apache Kafka® service.
+- `exampleTopic` is the name of the topic you are creating with tiered storage enabled.
+- The topic has the partition set to `2` and the replication factor set to `2`
 
-### Configure retention policies for a topic with tiered storage
+### Configure retention policies for a topic
 
-After enabling tiered storage, you can configure the retention policies
-for local storage:
+After enabling tiered storage, you can configure the retention policies for local storage:
 
 ```bash
 avn service topic-update \
-    --project demo-kafka-project \
-    --service-name demo-kafka-service \
-    --topic exampleTopic \
-    --local-retention-ms 100 \
-    --local-retention-bytes 10
+  --project demo-kafka-project \
+  --service-name demo-kafka-service \
+  --topic exampleTopic \
+  --local-retention-ms 100 \
+  --local-retention-bytes 10
 ```
 
-This command sets the local retention time to 100 milliseconds and the
-local retention size to 10 bytes for the topic named `exampleTopic` in
-the `demo-kafka-service` of the `demo-kafka-project`.
+This command sets the local retention time to 100 milliseconds and the local retention
+size to 10 bytes for the topic named `exampleTopic` in the `demo-kafka-service` of
+the `demo-kafka-project`.
 
 :::important
-If the values for `local_retention_bytes` and `local_retention_ms` are
-not set, they default to -2 or inherit the configuration from the
-service level.
-
-When set to -2, the retention in local storage will match the total
-retention. Consequently, data segments sent to remote storage are also
-retained locally. The remote storage will contain older data segments
-than the local storage, only if the total retention exceeds the local
+
+By default, `local_retention_bytes` and `local_retention_ms` are set to `-2` or
+inherit the configuration from the service level.
+
+When set to `-2`, the retention in local storage matches the total retention. In this
+scenario, the data segments sent to remote storage are also retained locally. Remote
+storage will contain older data segments only if the total retention exceeds the local
 retention.
 :::
+
+</TabItem>
+</Tabs>
+
+## (Optional) configure client-side parameter
+
+For optimal performance and reduced risk of broker interruptions when
+using tiered storage, it is recommended to update the client-side
+parameter `fetch.max.wait.ms` from its default value of 500 ms to 5000 ms.
```

Comment:
```suggestion
## (Only for Kafka 3.6.1) Configure client-side parameter

For optimal performance and reduced risk of broker interruptions when
using Tiered Storage, it is recommended to update the client-side
parameter `fetch.max.wait.ms` from its default value of 500 ms to 5000 ms.
Since 3.6.2 this consumer configuration is no longer needed.
Consider upgrading to 3.6.2 before enabling Tiered Storage.
```